### PR TITLE
v0.1 release

### PR DESCRIPTION
**Pull request by pzivich**
Initial release set as pull request for ease of code review tools

### CODE REVIEW

**Comment by pzivich** [comment]
For initial release, I should add a few more built-in estimating equations. Below are the ones I want to add (and will require corresponding tests)

- [ ] Ratio estimator
- [ ] Instrumental variable (4 part version?)
- [ ] Positive mean deviation
- [ ] Robust linear regression
- [ ] GLM
- [ ] GEE

**Comment by pzivich** [comment]
Adding 4PL, 3PL, and 2PL dose-response estimating equations. Further adding a 4PL EC estimating equation. It is just a transformation, so it generalized to each of the dose-response estimating equations. In the commits before this comment, but still needs to be tested

**Comment by pzivich** [comment]
Need to add a few further items to this release:

- [ ] Add weight optional argument for the built-in mean estimating equations
- [ ] Add weight optional argument for robust regression
- [ ] Add function to `MEstimator` to compute the CI for the user (easy to parse the `theta` and `variance`)

File: docs/Examples.rst [hunk 1]
```diff
@@ -200,40 +242,151 @@ Again, we can easily write these equations for ``delicatessen``,
...
+
+where :math:`k` indicates the bound, such that if :math:`Y_i>k` then :math:`k`, or :math:`Y_i<-k` then :math:`-k`,
+otherwise :math:`Y_i`. Below is the estimating equation translated into code
+
+.. code::
+
+    def psi_robust_mean(theta):
+        k = 3                          # Bound value
+        yr = np.where(y > k, k, y)     # Applying upper bound
+        yr = np.where(y < -k, -k, y)   # Applying lower bound
```
Comment (mwklose) [inline docs/Examples.rst:278]: yr is overwritten by itself, so example does not work as intended (error by ~0.02). 
Built-in estimating equation ([Line 144 of estimation_equations.py](https://github.com/pzivich/Delicatessen/blob/cb48d66b71fe606209683fb8f4d9d0875d53212a/delicatessen/estimating_equations.py#L144)) works as intended, so relative non-issue.

**Pull request merged**